docs/development.md

```diff
@@ -76,3 +76,21 @@ Check the issues or [raise a new one](https://github.com/webdriverio-community/w
 
 **[Help Wanted Issues](https://github.com/webdriverio-community/wdio-electron-service/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22)** \
 **[Good First Issues](https://github.com/webdriverio-community/wdio-electron-service/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22)**
+
+## Release
+
+Project maintainers can publish a release or pre-release of the npm package by manually running the [`Manual NPM Publish`](https://github.com/webdriverio-community/wdio-electron-service/actions/workflows/release.yml) GitHub Workflow. They will choose the release type to trigger a `major` , `minor`, or `patch` release following [Semantic Versioning](https://semver.org/).
+
+To publish a release, run the Workflow with default **Branch** `main` and **NPM Tag** `latest` and the appropriate **Release Type**. This will:
+
+* Create a Git tag
+* Create a GitHub Release
+* Publish to npm
+
+To publish a pre-release, also referred to as a test release, run the Workflow with the **NPM Tag** `next`. This will:
```

Comment:
Can we add a note which version bump to pick when making a pre-release?

Reply:
Ah, I missed that it'd be a `patch`. :+1: